docker-compose.yml

```diff
@@ -204,3 +204,79 @@ services:
       - ./config/sparql-authorization-wrapper:/config
     restart: always
     logging: *default-logging
+  ##############################################################################
+  # Consumers
+  ##############################################################################
+  besluiten-consumer-0:
+    image: lblod/delta-consumer:0.0.24
+    profiles:
+      - withConsumer
```

Comment:
this profiles stuff should go in the dev docker compose instead. We want it in the production version but if you want it in dev, you need to specify, else it will just be taking resources on your local machine without reason.